File: src/02_Manhattan_distance/manhattan_distance.cpp
#include <iostream>

using namespace std;

int main(void) {
    int x1, y1;
    int x2, y2;

    cin >> x1 >> y1;
    cin >> x2 >> y2;

    cout << abs(x2 - x1) + abs(y2 - y1) << endl;
    return 0;
}

File: src/02_Manhattan_distance/manhattan_distance.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1, x2 int
	var y1, y2 int
	var result float64

	fmt.Scanf("%d %d", &x1, &y1)
	fmt.Scanf("%d %d", &x2, &y2)

	result = math.Abs(float64((x2 - x1))) + math.Abs(float64((y2 - y1)))
	fmt.Println(int(result))
}